code/modules/mob/mob_helpers.dm

```diff
@@ -331,19 +331,86 @@
 	return returntext
 
 /proc/derpspeech(message, stuttering)
-	message = replacetext(message, " am ", " ")
-	message = replacetext(message, " is ", " ")
-	message = replacetext(message, " are ", " ")
-	message = replacetext(message, "you", "u")
-	message = replacetext(message, "help", "halp")
+	message = replacetext(message, "he", "eh") //he, she, the -> eh, seh, teh, etc
+	message = replacetext(message, "ies", "is") //monkis
+	message = replacetext(message, "you", "u") 
+	message = replacetext(message, "iou", "ou") //delicous
+	message = replacetext(message, "xc", "x") //exited
+	message = replacetext(message, "air", "er") //cher, her
+	message = replacetext(message, "uni", "uin")
+	message = replacetext(message, "dg", "g") //knowlege, 
+	message = replacetext(message, "tch", "ch") //bich
+	message = replacetext(message, "are", "ar")
+	message = replacetext(message, "pl", "pul")
+	message = replacetext(message, "oul", "ul")
+	message = replacetext(message, "ght", "gt")
+	message = replacetext(message, "gna", "ga")
+
+	if(prob(50)) //stuff we only want replaced this way SOME of the time, the alternative goes under ELSE
+		message = replacetext(message, "eau", "eu") //beutiful
+		message = replacetext(message, "ie", "ei")
+		message = replacetext(message, "tab", "tib") //tible, comfortible
+		message = replacetext(message, "'re", "re")
+		message = replacetext(message, "help", "halp")
+		message = replacetext(message, "wood", "woody")
+		message = replacetext(message, "ain", "ian")
+		if(prob(25)) //13%
+			message = replacetext(message, " am ", " ")
+			message = replacetext(message, " is ", " ")
+			message = replacetext(message, " are ", " ")
+	else //these prob checks are per sentence, so saying friend 100 times in one sentence will have all 100 look the same instead of mix and match
+		message = replacetext(message, "eau", "u")
+		message = replacetext(message, "friend", "fren")
+		message = replacetext(message, "tab", "teib") //teible, comforteible
+		message = replacetext(message, "you're", "your")
+		message = replacetext(message, "help", "helb")
+		message = replacetext(message, "y ", "i ")
+		message = replacetext(message, "y,", "i")
+		message = replacetext(message, "captain", "COMDOM")
+	if(prob(70))
+		var/regex/duplicate = new("(\\w)(?=\\1)", "g")
+		message = replacetext(message, duplicate, "") //duplicate letters into one letter, for all words
+	if(prob(40)) //occasionally replaces one word with umm.
+		var/list/words = splittext(message, " ")
+		words[rand(0, words.len)] = pick("um,", "umm,", "uh,", "uhh,")
+		message = jointext(words, " ")
+			
+	//specific words we want replaced every time, mainly references to the actual brain damage lines, so they stay consistent with brain damage regular speech
```

Comment:
if you want them replaced every time you should put this at the start of the proc, so the 2/3 letter replacements don't garble your input